doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -95,7 +95,7 @@ The following list summarizes the most important changes inherited from the upst
 Thread
 ------
 
-* Fixed ``otPlatCryptoPbkdf2GenerateKey`` API implementation to allow a fallback to legacy MbedTLS implementation instead of returning ``OT_ERROR_NOT_CAPABLE``.
+|no_changes_yet_note|
```

Comment:
There is one backported bugfix into 2.6.2 for Thread, please:

1) Remove the known issue for 2.6.2 and 2.7.0 (not sure why 2.7.0 is even there) -- KRKNWK-19036: High power consumption after parent loss.
2) We might want to add information about above e.g.: "Fixed issue with high power consumption after parent loss".

Reply:
Known issue will be removed in post release PR to main.